examples/kubernetes/manifests/registry.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: registry
+  labels:
+    app: registry
+    tier: registry
+spec:
+  type: NodePort
+  # type: LoadBalancer
+  ports:
+  - port: 5000
+    targetPort: 5000
+  selector:
+    app: registry
+    tier: registry
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: registry
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: registry
+        tier: registry
+    spec:
+      containers:
+      - name: registry
+        image: library/registry:2.6
+        command: ["/bin/sh", "/etc/docker/registry/init"]
```

Comment:
Yea, since making that comment I figured out that the certs were needed. Rather than doing it with an init script baked into the image, I ended up writing a simple helper script to create the certs and add them as a secret, then mounted the secret to both portus and registry. That makes it slightly more visible. A comment on what the script is doing and when you'd need or not need it would be very helpful.